scripts/win32.bat

```diff
@@ -128,12 +129,14 @@ Function GetTopLevelDrives()
 			Summary.Add "Size", TopLevelDrive.Size
 
 			Set Mountpoints = new List
+			Set MountpointObjects = new List
```

Comment:
Could the code be tidied up to use _just_ `MountpointObjects`, and then get rid of `Mountpoints` ? (to avoid the duplication). I'm afraid I don't know VBScript very well myself.